### PR TITLE
Package fix.20181205

### PR DESCRIPTION
**Pull request by fpottier**
### `fix.20181205`
Facilities for memoization and fixed points

---
* Homepage: https://gitlab.inria.fr/fpottier/fix
* Source repo: git+https://gitlab.inria.fr/fpottier/fix.git
* Bug tracker: francois.pottier@inria.fr

---
:camel: Pull-request generated by opam-publish v2.0.0

### CODE REVIEW

**Comment by fpottier** [comment]
My previous package did not install anything, perhaps because `fix.install` was constructed inside `src/` instead of at the root. I hope that this new archive and package description work, although I am not sure.

File: packages/fix/fix.20181205/opam [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  "François Pottier <francois.pottier@inria.fr>"
+]
+homepage: "https://gitlab.inria.fr/fpottier/fix"
+dev-repo: "git+https://gitlab.inria.fr/fpottier/fix.git"
+bug-reports: "francois.pottier@inria.fr"
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+]
+install: [
+  ["dune" "install" "-p" name]
```
Comment (kit-ty-kate) [inline packages/fix/fix.20181205/opam]: This rule is wrong. The `.install` file generated by the `dune build` is enough to install the package

**Comment by kit-ty-kate** [comment]
Your library looks wrong to me. I have a patch, give me 5 minutes

**Comment by kit-ty-kate** [comment]
Seems like gitlab.inria.fr doesn't allow me to fork your repo? Is that something you disabled?

**Comment by kit-ty-kate** [comment]
Fixed in https://gitlab.inria.fr/fpottier/fix/issues/1. Feel free to reopen or create a new PR with that fixed.

**Pull request closed without merging**